extension/chrome/src/js/background.js

```diff
@@ -70,6 +70,26 @@ const reloadLists = function() {
 	}).catch(console.error);
 };
 
+const reloadTemplate = function() {
+	extensionStorage.loadTemplateUrl()
+		.then(extensionStorage.loadTemplateFromUrl)
+		.then(template => {
+			if (!template) {
```

Comment:
might be useful to document that empty remote templates are not supported?

Reply:
or support empty templates

Reply:
just to make sure, this code is NOT called when the templateUrl is empty and/or for non-200 response, right?